pyproject.toml

```diff
@@ -85,11 +85,18 @@ include = [
 select = ["E", "F", "I", "N", "NPY", "NPY201"]
 
 [tool.cibuildwheel]
-build = "cp39-macosx* cp39-manylinux_x86_64 cp39-manylinux_aarch64"
+# By default, cibuildwheel builds for all supported Python versions available on
+# the CI runner. To build for a specific set of versions, uncomment the
+# "build" line below and edit the list.
+# build = "cp39-* cp310-* cp311-* cp312-* cp313-*"
```

Comment:
We only  build one wheel that is compatible with all Python versions. Please delete this comment.